SU-HSDO/suhumsci
# STN-599:  Bug - External link color refinement

# READY FOR REVIEW

## Summary
Refining link color using external classes and the hb-link class. External links were overriding hb-link classes if they are on the same link. The external link should match the link styling based on the theme/color pairing for hb-link. 

Jira Card: [STN-599](https://sparkbox.atlassian.net/browse/STN-599)

## Need Review By (Date)
10/27/20

## Urgency
medium

## Steps to Test
1. npm run test
2. To check the links that utilize both hb-link and the external link class, you will need to create a test page for these links.
Copy and paste this tex into a WYSIWYG editor under "Source" code:
```
<p><a class="hb-link ext" data-extlink="" href="http://drive.google.com/open?id=1Q-xLn8wGHmL8K8HghPJQh1x5hXSVD1k1">2018-19 Chronicle<span class="fa-ext extlink"><span aria-label="(link is external)" class="fas fa-arrow-right"></span></span></a></p>
<p><a class="hb-link ext" data-extlink="" href="https://drive.google.com/file/d/1L4i_O-2zq6_B1QZq2qy-ZBkKvCfeIen0/view">2017-18 Chronicl</a>e</p>
<p><a class="hb-link ext" data-extlink="" href="https://drive.google.com/file/d/0B8-EoqIvsi7-YTE1MmpfTHd6Wlk/view">2016-17 Chronicle<span class="fa-ext extlink"><span aria-label="(link is external)" class="fas fa-arrow-right"></span></span></a></p>
<p><a class="hb-link ext" data-extlink="" href="https://drive.google.com/file/d/0B8-EoqIvsi7-aC1BZS1GQmE5Tms/view">2015-16 Chronicle<span class="fa-ext extlink"><span aria-label="(link is external)" class="fas fa-arrow-right"></span></span></a></p>
<p><a class="hb-link ext" data-extlink="" href="https://drive.google.com/file/d/0B2WPuRUK2S_MSzBiOUVUS0FYRE0/view">2014-15 Chronicle<span class="fa-ext extlink"><span aria-label="(link is external)" class="fas fa-arrow-right"></span></span></a></p>
<p><a class="hb-link" href="/sites/creees/files/media/file/creees_chronicle_2013-14_for_web.pdf">2013-14 Chronicle</a></p>
<p><a class="ext" data-extlink="" href="https://creees.stanford.edu/sites/g/files/sbiybj11981/f/chronicle12-13.pdf?_nOipY3gxOc5bnyJfw2Rz5P6%E2%80%A6">2012-13 Chronicle<span class="fa-ext extlink"><span aria-label="(link is external)" class="fas fa-arrow-right"></span></span></a></p>
<p><a class="hb-link ext" data-extlink="" href="https://drive.google.com/file/d/0B8-EoqIvsi7-alJHb0pVVVF0YlE/view">2011-12 Chronicle<span class="fa-ext extlink"><span aria-label="(link is external)" class="fas fa-arrow-right"></span></span></a></p>
<p><a class="hb-link" href="https://creees-prod.stanford.edu/sites/creees/files/media/file/2010_creees_chronicle.pdf">2010-11 Chronicle</a></p>
```

3. You should see that the links that share both hb-link and the ext classes should use the hb-link color pairing and hover pairing consistently.
4. This should work for Colorful and Traditional theme options.


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
